optimum/onnxruntime/trainer.py

```diff
@@ -23,6 +23,12 @@
 import warnings
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Tuple, Type, Union
+from tqdm.auto import tqdm
+
+
+# Integrations must be imported before ML frameworks:
+from transformers.integrations import hp_params, is_fairscale_available  # isort: split
```

Comment:
useless references?

Reply:
this was coming from main when I resolved conflict

Reply:
but ideally after merge, this two lines should not appear in the diff.

Reply:
Will remove this to pass code quality check.

Reply:
Ok, I don't have push access. Can you use ruff styling with the command `make style` and remove the redundant dependencies? Thx!

Reply:
> Ok, I don't have push access. Can you use ruff styling with the command `make style` and remove the redundant dependencies? Thx!

I did make style and it formatted the trainer.py, but the CI still says it should be formatted.
is there a specific configuration for this?

Reply:
Hi @AdamLouly, yes the CI always uses the latest formatting tools. And whenever the team observe a failure of the check code quality CI,  we would fix it.

Your previous formatting issue could come from the fact that we recently switched from isort to ruff #760.  If you want to be more cautious, you can update your formatter with `pip install -U .[quality]` before `make style`.